core/jazz_es-kinesis-log-streamer/components/utils.js

```diff
@@ -136,96 +203,232 @@ function getApiLogsData(payload) {
-          data.message = data.message.replace(timestmp, "");
+// Function to get roleArn for specific accountId
+function getRolePlatformService(accountId, jsonConfig) {
+  var data = jsonConfig.config.AWS.ACCOUNTS;
+  var index = data.findIndex(x => x.ACCOUNTID == accountId);
+  return data[index].IAM.PLATFORMSERVICES_ROLEID;
+}
 
-          let guid = getSubInfo(data.message, config.PATTERNS.guid_regex, 0);
-          data.message = data.message.replace(guid, "");
+// Function to get configDB data
+function getConfigJson(config, token) {
+  return new Promise((resolve, reject) => {
+    var config_json_api_options = {
+      url: `${config.SERVICE_API_URL}${config.CONFIG_URL}`,
+      headers: {
+        "Content-Type": "application/json",
+        "Authorization": token
+      },
+      method: "GET",
+      rejectUnauthorized: false
+    };
+
+    request(config_json_api_options, (error, response, body) => {
+      if (error) {
+        reject(error)
+      } else {
+        if (response.statusCode && response.statusCode === 200) {
+          var responseBody = JSON.parse(body);
+          logger.debug("Response body of Config Json is: " +  JSON.stringify(responseBody));
+          resolve(responseBody.data)
+        } else {
+          logger.debug("Service not found for this service, domain, environment: ", JSON.stringify(config_json_api_options));
+          resolve([])
+        }
+      }
+    })
+  })
+}
 
-          data.message = data.message.replace(data.log_level, "");
+// Function to get accessToken
+function getToken(config) {
+  logger.debug("Inside getToken");
+  return new Promise((resolve, reject) => {
+    var svcPayload = {
+      uri: config.SERVICE_API_URL + config.TOKEN_URL,
+      method: 'post',
+      json: {
+        "username": config.SERVICE_USER,
+        "password": config.TOKEN_CREDS
+      },
+      rejectUnauthorized: false
+    };
+    request(svcPayload, (error, response, body) => {
+      if (response && response.statusCode === 200 && body && body.data) {
+        var authToken = body.data.token;
+        resolve(authToken);
+      } else {
+        var message = "";
+        if (error) {
+          message = error.message;
+        } else {
+          message = response.body.message
         }
-        data.message = data.message.trim();
-        let indexName = "applicationlogs";
-        let action = {
-          "index": {}
-        };
-        action.index._index = indexName;
-        action.index._type = "applicationlogs";
-        action.index._id = logEvent.id;
-
-        bulkRequestBody += [
-          JSON.stringify(action),
-          JSON.stringify(data),
-        ].join('\n') + '\n';
-      });
-    } else {
-      logger.error("invalid lambda logs event..: " + JSON.stringify(payload));
-    }
-    return bulkRequestBody;
-  } else {
-    return null;
-  }
+        logger.error(message);
+        reject({
+          "error": "Could not get authentication token for updating service catalog.",
```

Comment:
Incorrect message. We're not updating service catalog here. Suggestion - `Could not get authentication token`.

Reply:
Done.